scripts/AddHunter.cmake

```diff
@@ -23,7 +23,7 @@ macro(initialize_hunter)
   if(NOT HUNTER_ROOT AND "$ENV{HUNTER_ROOT}" STREQUAL "")
     set(HUNTER_ROOT "${ih_HUNTER_ROOT}" CACHE PATH "Hunter root dir")
   endif()
-  HunterGate(URL ${ih_URL} SHA1 ${ih_SHA1})
+  HunterGate(URL ${ih_URL} SHA1 ${ih_SHA1} LOCAL)
```

Comment:
Why? If it anything, it should be `${ih_UNPARSED_ARGUMENTS}`. That way it remains up to the user